docroot/modules/custom/layout_builder_custom/layout_builder_custom.module

```diff
@@ -282,7 +282,7 @@ function layout_builder_custom_form_alter(&$form, FormStateInterface $form_state
           // Limit the depth options to 2-4. We're removing the 1 option because
           // it causes issues with a discrepancy between the preview and what is
           // shown on the page after saving. Also, we consider it bad practice
-          // to shown the top-level menu again on a page.
+          // to show the top-level menu again on a page.
```

Comment:
Blah. This commit is supposed to say 'change shown to show'...

Reply:
And then I changed it again in [3babe1b](https://github.com/uiowa/uiowa/pull/4667/commits/3babe1b33f33c3ecb325ad3c68341c1747eeb148). TIL about gerund. /shrug